tests_integration/resources/empty-graph/README.md

```diff
@@ -0,0 +1,4 @@
+# cf-graph-countyfair (For Integration Tests)
+
+This repository is used in integration tests of [cf-scripts](https://github.com/regro/cf-scripts) to simulate the
```

Comment:
```suggestion
This repository is used in integration tests of [cf-scripts](https://github.com/regro/cf-scripts) to set up a clean initial state for the
```

Is that more accurate or did I get it wrong?

Reply:
That's good!